metabase/metabase
# Show more billing info on license and billing settings page

closes #38195

### Description

Implements the Store Info UI for #36367

### Remaining work
- [ ] validating that this works in the real world

### How to verify

- go to `/admin/settings/license`
- make sure you have `MB_PREMIUM_EMBEDDING_TOKEN` configured or have a token entered that is managed by the harbormaster store

### Demo

Full Data Preview (using mock data matching schema defined in [harbormaster spec](https://github.com/metabase/harbormaster/issues/4591)):
<img width="1491" alt="using-mock-data" src="https://github.com/metabase/metabase/assets/7104357/32e33146-0f2d-478f-a62f-580e63870d19">

Error state:
<img width="1493" alt="error-state" src="https://github.com/metabase/metabase/assets/7104357/857c9705-07ca-4b2f-860b-838e2023f730">

Invalid token state:
<img width="1490" alt="invalid-token" src="https://github.com/metabase/metabase/assets/7104357/73222c3d-8ab1-4765-98b8-9094b323808c">

Not managed by store:
<img width="1494" alt="not-managed-by-store" src="https://github.com/metabase/metabase/assets/7104357/ba72c459-ec95-46b4-84bb-15f61bf352d9">

### Checklist
